mondrian/pom.xml

```diff
@@ -13,14 +13,16 @@
   <description>Mondrian OLAP Engine</description>
   <url>http://mondrian.pentaho.com</url>
   <properties>
-    <project.version.minor>000</project.version.minor>
-    <driver.version>${project.version}</driver.version>
-    <project.version.major>8</project.version.major>
     <schema.version>3</schema.version>
+    <!-- The version should match the SQL Server version. Check the list here https://sqlserverbuilds.blogspot.com/ -->
```

Comment:
For future reference it might be worth briefly mentioning why the Mondrian version should match an SQL Server version greater or equal to SQL Server 2008 R2 (10.50.1600.1).

Reply:
Sorry, what I meant to ask for was to mention that this is done to support connections from PowerBI, so future changes have that into account.